libcontainerd/client_windows.go

```diff
@@ -299,7 +299,9 @@ func (clnt *client) createLinux(containerID string, checkpoint string, checkpoin
 		Owner:         defaultOwner,
 		TerminateOnLastHandleClosed: true,
 		HvRuntime: &hcsshim.HvRuntime{
-			ImagePath: `c:\program files\lcow`,
+			ImagePath:       `c:\program files\lcow`,
```

Comment:
While not a change in this commit, does it make sense to use the `ProgramFiles` environment variable instead of hard-coding, in the rare case it is something other than `C:\ProgramFiles` on the host?